test_config.placeholder.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I would probably advise against making this an array for this sample, since it still only supports one endpoint.

Reply:
I get that we ideally would make the schema consistent with the OMF samples, which require an array here, but I'd prefer to have a structure like:
```json
{
  "resource": "https://dat-b.osisoft.com",
  "endpoints": [
    { "resource": "https://dat-b.osisoft.com" }
  ]
}
```

Where the sample only has the parts of the schema that it needs; the OMF samples have the `endpoints` array to use for a list of endpoints, and the majority of the samples that only work against one endpoint just specify the info directly at the root. Having an array for a sample that doesn't support multiple seems misleading.

Reply:
I was conflicted about doing it like that since, on the one hand, it would be consistent across the samples, but on the other hand, it doesn't really make sense that there are different endpoints when only one is used for this sample. I think your suggestion of having a more flexible schema is the best option so I will rework that.